src/Sylius/Bundle/UserBundle/Command/DemoteUserCommand.php

```diff
@@ -49,21 +49,25 @@ protected function configure(): void
     protected function executeRoleCommand(InputInterface $input, OutputInterface $output, UserInterface $user, array $securityRoles): void
     {
         $error = false;
+        $successMessages = [];
 
         foreach ($securityRoles as $securityRole) {
             if (!$user->hasRole($securityRole)) {
-                $output->writeln(sprintf('<error>User "%s" didn\'t have "%s" Security role.</error>', $user->getEmail(), $securityRole));
+                $output->writeln(sprintf('<error>User "%s" doesn\'t have "%s" Security role.</error>', $user->getEmail(), $securityRole));
```

Comment:
I'm wondering if we shouldn't just throw an exception and return from the function

Reply:
Possibly, the current way has the advantage of showing every error to the user in cases where they try and remove multiple roles that aren't set, by throwing an exception and returning they will only see the first incorrect role.